### PR TITLE
Make select into error more user-friendly

### PR DESCRIPTION
**Pull request by JackieTien97**
Avoid display ugly `Task was cancelled` Exception to user.

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java [hunk 1]
```diff
@@ -490,7 +492,7 @@ private List<MeasurementPath> executeInternalCreateTimeseriesStatement(
           new IoTDBException(executionResult.status.getMessage(), statusCode));
     }
 
-    List<String> failedCreationList = new ArrayList<>();
+    Set<String> failedCreationList = new HashSet<>();
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java:495]: Rename to failedCreationMsgSet

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java [hunk 2]
```diff
@@ -502,12 +504,7 @@ private List<MeasurementPath> executeInternalCreateTimeseriesStatement(
     }
 
     if (!failedCreationList.isEmpty()) {
-      StringBuilder stringBuilder = new StringBuilder();
-      for (String message : failedCreationList) {
-        stringBuilder.append(message).append("\n");
-      }
-      throw new RuntimeException(
-          new MetadataException(String.format("Failed to auto create schema\n %s", stringBuilder)));
+      throw new SemanticException(new MetadataException(String.join(";, ", failedCreationList)));
```
Comment (MarcosZyk) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java:507]: Double separator, ```;``` and ```,``` ?

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java (hunk 1 above)
Comment (JackieTien97) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java]: done

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java (hunk 2 above)
Comment (JackieTien97) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/ClusterSchemaFetcher.java]: done

**Review by MarcosZyk — APPROVED** [review]

**Pull request merged**